azure-fs-aks/Program.fs

```diff
@@ -0,0 +1,131 @@
+﻿module Program
+
+open Pulumi.FSharp
+open Pulumi.Azure.Core
+open Pulumi.Azure.ContainerService
+open Pulumi.Azure.ContainerService.Inputs
+open Pulumi.Azure.AD
+open Pulumi.Azure.Network
+open Pulumi.Random
+open Pulumi.Tls
+
+[<RequireQualifiedAccess>]
+module Pulumi =
+    let createResourceGroup name = ResourceGroup(name)
```

Comment:
How about
```suggestion
    let createResourceGroup name = ResourceGroup name
```

Reply:
Nah, I like the extra `()` here to really indicate that it is an object that we are creating instead of just calling a function.

Reply:
Interesting... Should I also start doing so? Any links to guidelines?

Reply:
No links to guidelines, it is just my personal taste.

Reply:
@mikhailshilkov my reasoning is that I would use parentheses no argument (`unit`) and I would also use it when it is more than one argument to indicate that it can't be done partially so it make sense to me to be consistent and use it when it is only one argument as well.